src/Containers/TableFeature/TableFeature.stories.tsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+<<<<<<< HEAD
```

Comment:
Merge issue this code won't run, make sure you test your work!

Reply:
hi, @ralph-dev files are now updated issue has been fixed.